src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -0,0 +1,8 @@
+package frc.robot.subsystems.elevator;
+
+public class ElevatorConstants {
+    public static final double MECHANISM_WIDTH = 0;
+    public static final double MECHANISM_HEIGHT = 0;
+    public static final double ROOT_X = 0;
+    public static final double ROOT_Y = 0;
```

Comment:
What do this mean?

Reply:
263c505cbffebc4fd331bcd5570e1bf8f2d7c98c